src/sagemaker/tensorflow/README.rst

```diff
@@ -460,175 +144,90 @@ you can specify these as keyword arguments.
-fed with the evaluation dataset defined by ``eval_input_fn``. See `Creating train_input_fn and eval_input_fn functions`_.
-
-For more information on training and evaluation process, see `tf.estimator.train_and_evaluate <https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/python/estimator/training.py#L256>`_.
-
-For more information on fit, see `SageMaker Python SDK Overview <#sagemaker-python-sdk-overview>`_.
-
-TensorFlow Serving models
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-After your training job is complete in SageMaker and the ``fit`` call ends, the training job
-will generate a `TensorFlow SavedModel <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md>`_
-bundle ready for deployment. Your model will be available in S3 at the ``output_path`` location
-that you specified when you created your ``sagemaker.tensorflow.TensorFlow`` estimator.
-
-Restoring from checkpoints
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-While your training job is executing, TensorFlow will generate checkpoints and save them in the S3
-location defined by ``checkpoint_path`` parameter in the ``TensorFlow`` constructor.
-These checkpoints can be used to restore a previous session or to evaluate the current training using ``TensorBoard``.
-
-To restore a previous session, you just need to create a new ``sagemaker.tensorflow.TensorFlow``
-estimator pointing to the previous checkpoint path:
-
-.. code:: python
-
-  previous_checkpoint_path = 's3://location/of/my/previous/generated/checkpoints'
-
-  tf_estimator = TensorFlow('tf-train.py', role='SageMakerRole',
-                            checkpoint_path=previous_checkpoint_path
-                            training_steps=10000, evaluation_steps=100,
-                            train_instance_count=1, train_instance_type='ml.p2.xlarge',
-                            framework_version='1.10.0')
-  tf_estimator.fit('s3://bucket/path/to/training/data')
+Required:
 
+-  ``entry_point (str)`` Path (absolute or relative) to the Python file which
+   should be executed as the entry point to training.
+-  ``role (str)`` An AWS IAM role (either name or full ARN). The Amazon
+   SageMaker training jobs and APIs that create Amazon SageMaker
+   endpoints use this role to access training data and model artifacts.
+   After the endpoint is created, the inference code might use the IAM
+   role, if accessing AWS resource.
+-  ``train_instance_count (int)`` Number of Amazon EC2 instances to use for
+   training.
+-  ``train_instance_type (str)`` Type of EC2 instance to use for training, for
+   example, 'ml.c4.xlarge'.
+- ``training_steps (int)`` Perform this many steps of training. ``None``, means train forever.
+- ``evaluation_steps (int)`` Perform this many steps of evaluation. ``None``, means
+  that evaluation runs until input from ``eval_input_fn`` is exhausted (or another exception is raised).
 
-Running TensorBoard
-^^^^^^^^^^^^^^^^^^^
+Optional:
```

Comment:
All these parameters are available for script mode as well, which makes this part confusing. I would just remove any mention to legacy mode here and explain how these fields work with script mode.